### ISSUE TITLE
Opening Eclipse marketplace crashes dbeaver 

### ISSUE THREAD

**Issue by buddemat** [dbeaver/dbeaver#9225]
#### System information: 
- Linux (Ubuntu 20.04 LTS)
- DBeaver CE Standalone 7.1.2.202007051953
- Java version openjdk 11.0.7 2020-04-14 (error log prints "java.version=1.8.0_252")
- Additional extensions: 
    - CodeTogether Collaboration (2.0.0)
    - DevSyle (1.11.0)
    - Marketplace Client (1.8.3)

#### Connection specification:
- Database name and version (irrelevant)
- Driver name (irrelevant)
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  (irrelevant)

#### Describe the problem you're observing:
When trying to open the Eclipse Marketplace via the "Help" menu, dbeaver crashes.

#### Steps to reproduce, if exist:
All I did was to install the Eclipse Marketplace via the "Help -> Install New Software" menu

#### Include any warning/errors/backtraces from the logs
Plug-in: com.genuitec.eclipse.theming.core
Severity: Error
Message: `<null>`
Exception Stack Trace:
```
Unexpected token RIGHT SQUARE(]) at position 0.
	at org.json.simple.parser.JSONParser.parse(JSONParser.java:257)
	at org.json.simple.parser.JSONParser.parse(JSONParser.java:92)
	at com.genuitec.eclipse.theming.core.services.ImageReplacementService.loadUnknownImages(ImageReplacementService.java:91)
	at com.genuitec.eclipse.theming.core.services.ImageReplacementService.<init>(ImageReplacementService.java:74)
	at com.genuitec.eclipse.theming.core.services.ImageReplacementService.<clinit>(ImageReplacementService.java:38)
	at com.genuitec.eclipse.theming.core.internal.images.URLImageDescriptorWeaver.weave(URLImageDescriptorWeaver.java:23)
	at com.genuitec.eclipse.theming.core.AbstractWeaverExtension.weave(AbstractWeaverExtension.java:51)
	at com.genuitec.eclipse.patches.internal.InstrumentationManager$WeavingExtension.weave(InstrumentationManager.java:424)
	at com.genuitec.eclipse.patches.internal.InstrumentationManager.weave(InstrumentationManager.java:327)
	at com.genuitec.eclipse.patches.internal.InstrumentationWeaver.weave(InstrumentationWeaver.java:15)
	at org.eclipse.osgi.internal.weaving.WovenClassImpl.call(WovenClassImpl.java:175)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.notifyHookPrivileged(ServiceRegistry.java:1381)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.notifyHooksPrivileged(ServiceRegistry.java:1358)
	at org.eclipse.osgi.internal.weaving.WovenClassImpl.callHooks(WovenClassImpl.java:270)
	at org.eclipse.osgi.internal.weaving.WeavingHookConfigurator.processClass(WeavingHookConfigurator.java:71)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.processClass(ClasspathManager.java:736)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.defineClass(ClasspathManager.java:707)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findClassImpl(ClasspathManager.java:640)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClassImpl(ClasspathManager.java:608)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClassImpl(ClasspathManager.java:588)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:567)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:346)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:398)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:477)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:171)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	at org.eclipse.ui.internal.WorkbenchImages.declareImage(WorkbenchImages.java:106)
	at org.eclipse.ui.internal.WorkbenchImages.declareImages(WorkbenchImages.java:116)
	at org.eclipse.ui.internal.WorkbenchImages.initializeImageRegistry(WorkbenchImages.java:460)
	at org.eclipse.ui.internal.WorkbenchImages.getImageRegistry(WorkbenchImages.java:434)
	at org.eclipse.ui.internal.Workbench$17.runWithException(Workbench.java:1629)
	at org.eclipse.ui.internal.StartupThreading$StartupRunnable.run(StartupThreading.java:36)
	at org.eclipse.swt.widgets.Synchronizer.syncExec(Synchronizer.java:236)
	at org.eclipse.ui.internal.UISynchronizer.syncExec(UISynchronizer.java:133)
	at org.eclipse.swt.widgets.Display.syncExec(Display.java:5795)
	at org.eclipse.ui.internal.StartupThreading.runWithoutExceptions(StartupThreading.java:94)
	at org.eclipse.ui.internal.Workbench.init(Workbench.java:1625)
	at org.eclipse.ui.internal.Workbench.runUI(Workbench.java:2793)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:646)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:557)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:154)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.start(DBeaverApplication.java:233)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:401)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:657)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:594)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1447)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1420)
```
Session Data:
```
eclipse.buildId=unknown
java.version=1.8.0_252
java.vendor=Private Build
BootLoader constants: OS=linux, ARCH=x86_64, WS=gtk, NL=en_US
Command-line arguments:  -os linux -ws gtk -arch x86_64
```

**Comment by kseniiaguzeeva** [comment]
Please tell what exactly do you write in **Work with** field?
![dbeaver_XgN3jkcoTf](https://user-images.githubusercontent.com/51405061/87160752-21898900-c2cc-11ea-9778-1c481f3689e8.png)

**Comment by buddemat** [comment]
I just selected "All available sites"

![Screenshot from 2020-07-13 10-38-48](https://user-images.githubusercontent.com/6584684/87283606-c0dd9480-c4f5-11ea-8595-abb161238f30.png)

Installation seemed successful, too. The crash occurs afterwards when I try to open the marketplace from the menu

![Screenshot from 2020-07-13 10-42-18](https://user-images.githubusercontent.com/6584684/87284418-a2c46400-c4f6-11ea-9614-fd9046bb24bf.png)

When I click, dbeaver crashes.

**Comment by serge-rider** [comment]
Seems to be a bug in Eclipse + Darkest Dark + specific GTK version.

**Comment by serge-rider** [comment]
Probably fixed in #10987
